public/app/plugins/datasource/cloud-monitoring/components/AnnotationsHelp.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export const AnnotationsHelp = () => {
   return (
-    <div className="gf-form grafana-info-box alert-info">
```

Comment:
This is not from the Loki data source, so maybe either revert the change or update the PR title and description please. Thank you!